src/SFA.DAS.Forecasting.Application/Shared/Services/EmployerDatabaseService.cs

```diff
@@ -31,6 +32,29 @@ public class EmployerDatabaseService : BaseRepository, IEmployerDatabaseService
             _logger = logger;
         }
 
+        public async Task<IList<long>> GetAccountIds(string payrollYear, short payrollMonth)
+        {
+            var result = await WithConnection(async c =>
+            {
+                var parameters = new DynamicParameters();
+                parameters.Add("@payrollYear", payrollYear, DbType.String);
+                parameters.Add("@payrollMonth", payrollMonth, DbType.Int16);
+                var sql = @"Select distinct
+	                    ldt.AccountId
+                        from [employer_financial].[TransactionLine] tl
+                        join [employer_financial].GetLevyDeclaration ldt on tl.SubmissionId = ldt.SubmissionId
```

Comment:
I think i would be tempted to join on the levydeclaration table rather than the view if you are just getting the payrollyear and month. You dont need all the extra stuff that is in the view